### DISCUSSION TITLE
Access denied to .local-chromium when using .net 8 image

### DISCUSSION THREAD

**Discussion by atmoraes1** [hardkoded/puppeteer-sharp#2519] [General]
When using .net 8 images Puppeteer does not have access to the chromium folder as it runs in user mode.

![image](https://github.com/hardkoded/puppeteer-sharp/assets/139466904/dec6626e-03ef-4dfa-a604-b0c961dacb27)

Is there a recommended solution for that issue or do we just need to create the folder with the required permissions?

Using the last version.

**Comment by kblok** [comment]
When you say `using .net8`, does it mean that it works with .net6 or 7?

**Comment by atmoraes1** [comment]
Yes, because previous images run in root mode, so the profile folder has all the required permissions.

**Comment by kblok** [comment]
So it's more something you need to fix in the dockerfile right?

**Comment by atmoraes1** [comment]
Yeap, just wanted to know if there is a recommended approach to that or if one just need to give the specific permissions to the directory.
